libraries/cli/include/cli/config_jsons/mainnet.json

```diff
@@ -211,12 +211,12 @@
         "dpos": {
           "delegation_delay": "0x5",
           "delegation_locking_period": "0x5",
-          "eligibility_balance_threshold": "0x186A0",
-          "vote_eligibility_balance_step": "0x186A0",
-          "validator_maximum_stake":"0x84595161401484A000000",
-          "minimum_deposit":"0x0",
-          "max_block_author_reward": "0xa",
-          "commission_change_delta":"0x0",
+          "eligibility_balance_threshold": "0x69E10DE76676D0800000",
+          "vote_eligibility_balance_step": "0x3635C9ADC5DEA00000",
+          "validator_maximum_stake":"0x422CA8B0A00A4250000000",
+          "minimum_deposit":"0x3635C9ADC5DEA00000",
+          "max_block_author_reward": "0x5",
+          "commission_change_delta":"0x1F4",
           "commission_change_frequency":"0x0",
```

Comment:
to change this, we need to verify if we calculate avg block time correctly.

now we define it as 2.5 *  lambda so for testnet where lambda = 1500m it should be 5.25s but in reality based on explorer it is 3-4 seconds